src/brackets.config.json

```diff
@@ -2,19 +2,19 @@
     "config"                :
     {
         "app_title"                  : "Brackets",
-        "app_name_about"             : "Brackets",
+        "app_name_about"             : "Brackets Continued",
         "about_icon"                 : "styles/images/brackets_icon.svg",
-        "how_to_use_url"             : "https://github.com/adobe/brackets/wiki/How-to-Use-Brackets",
-        "support_url"                : "https://github.com/adobe/brackets/wiki/Troubleshooting",
-        "suggest_feature_url"        : "https://github.com/adobe/brackets/wiki/Suggest-a-Feature",
-        "get_involved_url"           : "https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md",
-        "glob_help_url"              : "https://github.com/adobe/brackets/wiki/Using-File-Filters",
-        "release_notes_url"          : "https://github.com/adobe/brackets/wiki/Release-Notes",
+        "how_to_use_url"             : "https://github.com/brackets-cont/brackets/wiki/How-to-Use-Brackets",
+        "support_url"                : "https://github.com/brackets-cont/brackets/wiki/Troubleshooting",
+        "suggest_feature_url"        : "https://github.com/brackets-cont/brackets/wiki/Suggest-a-Feature",
+        "get_involved_url"           : "https://github.com/brackets-cont/brackets/blob/master/CONTRIBUTING.md",
+        "glob_help_url"              : "https://github.com/brackets-cont/brackets/wiki/Using-File-Filters",
+        "release_notes_url"          : "https://github.com/brackets-cont/brackets/wiki/Release-Notes",
         "homepage_url"               : "http://brackets.io",
         "twitter_url"                : "https://twitter.com/brackets",
```

Comment:
Pretty minor but I heard brackets people won't give us access to this Twitter account so somebody made https://twitter.com/BracketsCont